### ISSUE TITLE
Security: -allowed_nets does not work if we do not set -allowed_sender option.

### ISSUE THREAD

**Issue by Daniel-Marynicz** [grafana/smtprelay#21]
in https://github.com/grafana/smtprelay/blob/d399838e7fd1cfc72626682f4b3897616ec70105/main.go#L66  we have 

```
return nil
```
Probably in https://github.com/decke/smtprelay  this is resolved.

**Comment by hairyhenderson** [comment]
Thanks for the report @Daniel-Marynicz, and apologies for the radio silence (this repo was without any dedicated maintainers until recently).

I'll take a look at this soon.

**Comment by hairyhenderson** [comment]
> Probably in https://github.com/decke/smtprelay this is resolved.

Indeed, it looks like this was handled as part of https://github.com/decke/smtprelay/pull/18 based on https://github.com/grafana/smtprelay/pull/4#discussion_r593717901
